src/content/en/updates/2018/07/page-lifecycle-api.md

```diff
@@ -0,0 +1,962 @@
+        <strong>Note:</strong>
+        a blur event does not necessarily imply a state change. In many cases a
+        blur event means a user is switching focus from one form element to
+        another.
+      </aside>
+      <p>
+        <strong>Possible previous states:</strong><br>
+        <a href="#state-active">active</a>
+      </p>
+      <p>
+        <strong>Possible current states:</strong><br>
+        <a href="#state-passive">passive</a>
+      </p>
+    </td>
+  </tr>
+  <tr>
+   <td><a href="https://developer.mozilla.org/en-US/docs/Web/Events/visibilitychange">
+   visibilitychange</a></td>
+   <td>
+      <p>The document's visibilityState value has changed. This can happen when a
+      user navigates to a new page, switches tabs, closes a tab, minimizes or
+      closes the browser, or switches apps on mobile operating systems.</p>
+      <p>
+        <strong>Possible previous states:</strong><br>
+        <a href="#state-passive">passive</a><br>
+        <a href="#state-hidden">hidden</a>
+      </p>
+      <p>
+        <strong>Possible current states:</strong>
+        <a href="#state-passive">passive</a><br>
+        <a href="#state-hidden">hidden</a>
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td><a href="https://wicg.github.io/page-lifecycle/spec.html#sec-api">freeze</a>
+      <span style="color:red">*</td>
+    <td>
+      <p>The page has just been frozen. Any
+      <a href="https://wicg.github.io/page-lifecycle/spec.html#html-task-source-dfn">
+      freezable</a> task in the page's task queues will not be started.</p>
+      <p>
+        <strong>Possible previous states:</strong><br>
+        <a href="#state-hidden">hidden</a>
+      </p>
+      <p>
+        <strong>Possible current states:</strong><br>
+        <a href="#state-frozen">frozen</a>
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td><a href="https://wicg.github.io/page-lifecycle/spec.html#sec-api">
+    resume</a> <span style="color:red">*</td>
+    <td>
+      <p>The browser has resumed a frozen page.</p>
+      <p>
+        <strong>Possible previous states:</strong><br>
+        <a href="#state-frozen">frozen</a>
+      </p>
+      <p>
+        <strong>Possible current states:</strong><br>
+        <a href="#state-active">active</a> (if followed by the
+        <a href="https://developer.mozilla.org/en-US/docs/Web/Events/pageshow">
+        pageshow</a> event)<br>
+        <a href="#state-passive">passive</a> (if followed by the
+        <a href="https://developer.mozilla.org/en-US/docs/Web/Events/pageshow">
+        pageshow</a> event)<br>
+        <a href="#state-hidden">hidden</a>
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td> <a href="https://developer.mozilla.org/en-US/docs/Web/Events/pageshow">pageshow</a></td>
+    <td>
+      <p>A session history entry is being traversed to.</p>
+      <p>This could be either a brand new page load or a page taken from the
+      page navigation cache. If the page was taken from the page navigation
```

Comment:
It's a general term I made up because I didn't want to use the Webkit version (Page Cache) or the Firefox version (Back-Forward Cache or bfcache), but I agree it could probably use some clarity.

Maybe I'll add it to the FAQs and link to that entry any time I use it.

Reply:
Done.

Reply:
I won 't hold up because of this, but best practice is to give at least enough to make sense of the thing at the first occurrence. Then the 'more information' link is for more depth, not the basics.